doc/taoQtiTestPreviewer.yaml

```diff
@@ -0,0 +1,247 @@
+openapi: 3.0.1
+info:
+  title: taoTestPreview API
+  description: Stands for TAO test Previewer API.
+  version: 1.0.0
+servers:
+  - url: '{server}'
+    variables:
+      server:
+        default: https://community.docker.localhost/
+security:
+  - cookieAuth: []
+paths:
+  /taoQtiTestPreviewer/TestPreviewer/init:
+    get:
+      parameters:
+        - in: query
+          name: testUri
+          schema:
+            type: string
+          required: true
+          example: 'https%3A%2F%2Fcommunity.docker.localhost%2Fontologies%2Ftao.rdf%23i5efb42fced1ce7064796c08b38ee176'
+          description: Url encoded test identifier
+
+      summary: Returns a test map
+      responses:
+        '200':
+          $ref: '#/components/responses/InitResponse'
+        '500':
```

Comment:
either this is wrong or the line 223

Reply:
are we providing just 200 and 500 ? no 400 about a bad request?

Reply:
Line 223 is just `example: 400`, but yes. The server sends a 400 :) 

Fixed